kie-server-parent/kie-server-tests/kie-server-integ-tests-custom-extension/kie-server-integ-tests-custom-extension-test/pom.xml

```diff
@@ -15,8 +15,19 @@
   <properties>
     <java.module.name>org.kie.server.itests.custom.extension.test</java.module.name>
     <org.drools.server.ext.disabled>false</org.drools.server.ext.disabled>
+    <org.kie.prometheus.server.ext.disabled>false</org.kie.prometheus.server.ext.disabled>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.kie.server</groupId>
+        <artifactId>kie-server-integ-tests-custom-extension-prometheus</artifactId>
+        <version>7.26.0-SNAPSHOT</version>
```

Comment:
Please move this dependency to kie-server-tests pom.xml, also use <version>${version.org.kie}</version> as artifact version.